### PR TITLE
set graphbinary as default; fix tests

### PR DESCRIPTION
**Pull request by vkagamlyk**
*Issue #, if available:*
https://bitquill.atlassian.net/browse/AN-1165

*Description of changes:*
[TINKERPOP-2693] Finish graphbinary support and enable it by default in gremlin-python

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by lyndonbauto** [comment]
Should we target 3.5-dev?

**Comment by vkagamlyk** [comment]
> Should we target 3.5-dev?

I'm not sure, in [ticket](https://issues.apache.org/jira/browse/TINKERPOP-2723) mentioned 3.6

File: gremlin-python/src/main/python/gremlin_python/driver/serializer.py [hunk 1]
```diff
@@ -24,6 +24,7 @@
 import uuid
 import io
 
+from gremlin_python.process import traversal
```
Comment (xiazcy) [inline gremlin-python/src/main/python/gremlin_python/driver/serializer.py]: Is this actually used? I don't see any reference to this in the changes.

**Comment by lyndonbauto** [comment]
I think we should make graphbinary default in 3.6 but type support could go to 3.5

File: gremlin-python/src/main/python/tests/driver/test_client.py [hunk 2]
```diff
@@ -47,11 +48,12 @@ def test_connection(connection):
 
 def test_client_message_too_big(client):
     try:
-        client = Client("http://localhost", 'g', max_content_length=1024)
-        client.submit("1+1").all().result()
+        client = Client('ws://localhost:45940/gremlin', 'g', max_content_length=1024)
```
Comment (xiazcy) [inline gremlin-python/src/main/python/tests/driver/test_client.py]: I see that this url `ws://localhost:45940/gremlin` is repeatedly used throughout the test suite, perhaps we should abstract it to a constant so it's easier to change in the future?

File: gremlin-python/src/main/python/gremlin_python/driver/serializer.py (hunk 1 above)
Comment (vkagamlyk) [reply gremlin-python/src/main/python/gremlin_python/driver/serializer.py]: not used, removed

File: gremlin-python/src/main/python/tests/driver/test_client.py (hunk 2 above)
Comment (vkagamlyk) [reply gremlin-python/src/main/python/tests/driver/test_client.py]: done in https://github.com/lyndonbauto/tinkerpop/pull/118/commits/7d7464e81c5c6c3888fc6e9ea1c1a6633f6f0eed

**Pull request closed without merging**